### PR TITLE
docs: document the Host Rules UI for Mend-hosted Apps

### PR DESCRIPTION
**Pull request by justo-mend**
## Changes

Added docs for Host Rules UI in the Developer Portal

## Context

New UI has been added in the Developer Portal for creating Host Rules.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — CHANGES_REQUESTED** [review]
This is not a good example to use like this, because:
1. If any github user applies this example, it can break break the(ir) app by overriding the app token
2. It should have a match host for github.com

If you want to show a github host rule for those on bitbucket/azdo who need it for fetching, then add more description making that part clear.

File: docs/usage/mend-hosted/.pages [hunk 1]
```diff
@@ -1,5 +1,5 @@
 title: Mend-hosted Apps
 nav:
   - 'Configuration': 'hosted-apps-config.md'
-  - 'App Secrets': 'app-secrets.md'
+  - 'Secrets and Host Rules': 'app-secrets.md'
```
Comment (rarkins) [inline docs/usage/mend-hosted/.pages]:
```suggestion
  - 'Credentials': 'credentials.md'
```

Rename the file. Is it linked from anywhere external?

File: docs/usage/mend-hosted/app-secrets.md [hunk 2]
```diff
@@ -30,7 +30,8 @@ To add a secret for the Mend cloud app:
 
    ![Credentials settings page](../assets/images/app-settings/app-credentials.png)
 
-4. Reference the secret from Renovate config files inside the repo.
+4. Use can reference the secret from Renovate config files inside the repo.
```
Comment (rarkins) [inline docs/usage/mend-hosted/app-secrets.md]:
```suggestion
4. Reference the secret from Renovate config files inside the repo.
```

File: docs/usage/mend-hosted/app-secrets.md [hunk 3]
```diff
@@ -43,6 +44,19 @@ To add a secret for the Mend cloud app:
    }
    ```
 
+### Adding a host rule
```
Comment (rarkins) [inline docs/usage/mend-hosted/app-secrets.md]:
```suggestion
### Adding a host rule through the UI

You can centrally add/configure Host Rules through the Mend UI as an alternative to including them in Renovate presets.
```

File: docs/usage/mend-hosted/app-secrets.md [hunk 4]
```diff
@@ -43,6 +44,19 @@ To add a secret for the Mend cloud app:
...
+
+1. Open the _Credentials_ section of the settings page for the relevant Org or Repo.
+2. Select `ADD HOST RULE` to open the "Add a Host Rule" dialog box.
+
+   ![Add Host Rule](../assets/images/app-settings/add-host-rule.png)
+
+3. Fill out the details for your host rule.
+
+   As an example, if you are a Bitbucket or Azure DevOps user, and you want to specify the GITHUB_TOKEN to fetch release notes, you could create a host rule like this:
```
Comment (rarkins) [inline docs/usage/mend-hosted/app-secrets.md]:
```suggestion
   As an example, if you are a Bitbucket or Azure DevOps user, and you want to specify a github.com token to fetch release notes and enable github-based datasources, you could create a host rule like this:
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/usage/mend-hosted/.pages (hunk 1 above)
Comment (justo-mend) [reply docs/usage/mend-hosted/.pages]: I'll check for external references.

**Comment by rarkins** [comment]
This is ready to raise to the main repo, if main tests pass

**Comment by justo-mend** [comment]
PR created to merge this branch into the main renovatebot repo.
Closing this PR.

**Pull request closed without merging**